proposals/nomination-schedule.md

```diff
@@ -0,0 +1,24 @@
+# Proposed Nomination Schedule
+
+
+
+| Event                          | Time Taken  | What's involved |
+| :------------------------------ |:-----------| :---------------|
+| Marketing of board nominations | 1 week      | - Raise awareness of the board nominations coming up. <br/> - Share requirements for being eligible for being a board member. <br/> - Share the commitments and responsibilities of being a board member.<br/> - Share how to nominate yourself or someone else for the board. <br/> - Share how and why to become a member of the .NET Foundation.  |
```

Comment:
```suggestion
| Marketing | 2+ months | - Announce the nomination schedule at Build end of May. <br/> - Comms/PR plan (become a member campaign). <br/> - Co-marketing plan with partners & corporate sponsors.  <br/> - Create promotion assets and copy.  |
| Promotion of board nominations | 2+ weeks      | - Raise awareness of the board nominations coming up. <br/> - Share requirements for being eligible for being a board member. <br/> - Share the commitments and responsibilities of being a board member.<br/> - Share how to nominate yourself or someone else for the board. <br/> - Share how and why to become a member of the .NET Foundation.  |
```

Reply:
Added more runway for marketing. I think we'd want to capitalize on the PR from Build to start a membership drive and announce the nomination schedule early.